src/platform/user/authorization/containers/MHVApp.jsx

```diff
@@ -4,11 +4,11 @@ import { connect } from 'react-redux';
 import { withRouter } from 'react-router';
 import appendQuery from 'append-query';
 
+// eslint-disable-next-line deprecate/import
 import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
+// eslint-disable-next-line deprecate/import
 import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
```

Comment:
I am temporarily disabling the `AlertBox` and `LoadingIndicator` deprecation errors because this is outside of the scope of the work here. We will be addressing replacing these with web components in the following issues:

- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1918
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1917